src/components/Main/Schedule/NewScheduleForm.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useState } from 'react';
+import { FormField } from '../../shared/FormField';
+
+import { TextField, Select } from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+import FormHelperText from '@material-ui/core/FormHelperText';
+// import InfoIcon from '@material-ui/icons/Info';
+
+// import ReactTooltip from 'react-tooltip';
+import { Tooltip } from '../../shared/Tooltip';
+
+// TODO hook up teams with data from DB
+
+interface NewScheduleFormProps {
+  onSubmit: (
+    scheduleTitle: string,
+    startDate: string,
+    endDate: string,
+    view: string,
+    team: number,
+  ) => void;
+  onClose: () => void;
+}
+
+export const NewScheduleForm = ({ onSubmit, onClose }: NewScheduleFormProps) => {
+  const [scheduleTitle, setScheduleName] = useState('');
+  const [startDate, setStartDate] = useState(toDateString(new Date()));
+  const [endDate, setEndDate] = useState(toDateString(new Date()));
+  const view = 'weekly';
+  const [team, setTeam] = useState<number>(0);
+  const [errorMessage, setErrorMessage] = useState('');
+  const classes = useStyles();
+
+  //needed to format date so that the date picker can display it properly
+  function toDateString(date: Date): string {
+    return `${date.getFullYear()}-${date.getMonth()}-${date.getDate()}`;
+  }
+
+  function NewlineText(props: { text: string }) {
+    const text = props.text;
+    const newText = text.split('\n').map((str) => <p>{str}</p>);
+    return <div>{newText}</div>;
+  }
+
+  function onSubmitForm() {
+    setErrorMessage('');
+    let msg: string = '';
+    if (scheduleTitle.length === 0 || scheduleTitle.length >= 32)
+      msg = '-Title must be not be blank and be under 32 characters long'; //line breaks don't work ahhh
+    if (new Date(endDate) <= new Date(startDate)) msg += '\n-Invalid Date Range';
+    if (team === 0) msg += '\n-Please assign a team to this schedule';
+    if (msg.length > 0) return setErrorMessage(msg);
```

Comment:
Having trouble making line breaks work? Does using `<br/>` work?
```suggestion
    if (scheduleTitle.length === 0 || scheduleTitle.length >= 32)
      msg = '-Title must be not be blank and be under 32 characters long'; //line breaks don't work ahhh
    if (new Date(endDate) <= new Date(startDate)) msg += '<br/>-Invalid Date Range';
    if (team === 0) msg += '<br/>-Please assign a team to this schedule';
    if (msg.length > 0) return setErrorMessage(msg);
```